usr/local/www/firewall_aliases_edit.php

```diff
@@ -68,12 +72,25 @@
 
 if (!is_array($config['aliases']['alias']))
 	$config['aliases']['alias'] = array();
+
 $a_aliases = &$config['aliases']['alias'];
 
 $tab = $_REQUEST['tab'];
 
-if($_POST)
+if($_POST) {
 	$origname = $_POST['origname'];
+	
+// Zap any blank addresses in the array
```

Comment:
this can be done much simpler; use `$_POST['address'] = array_filter($_POST['address']);` for the same effect